test/e2e/e2e_docker_helper.go

```diff
@@ -211,3 +212,14 @@ func setupByoDockerHost(ctx context.Context, clusterConName, byoHostName, namesp
 
 	return output, byohost.ID, err
 }
+
+func setControlPlaneIP(ctx context.Context, dockerClient *client.Client) {
```

Comment:
Add comment on what this function is doing. Because it is doing some specific hard coding to 151. So good to know why.
Something along the lines of
`The ControlPlaneEndpoint is a static IP that is in the hosts' subnet but outside of DHCP range. We believe 151 is a pretty high number and we have < 10 containers being spun up, so we can safely use this IP for the ControlPlaneEndpoint`

Reply:
Added comments.